test/Deserialization.purs

```diff
@@ -144,27 +150,8 @@ suite = do
           newTransactionUnspentOutputFromBytes utxoFixture1 >>=
             convertUnspentOutput
         res `shouldEqual` utxoFixture1'
-    group "Transaction" do
-      test "deserialization is inverse to serialization #1" do
-        let input = txFixture1
-        serialized <- liftEffect $ TS.convertTransaction input
-        let expected = TD.convertTransaction serialized
-        pure input `shouldEqual` hush expected
-      test "deserialization is inverse to serialization #2" do
-        let input = txFixture2
-        serialized <- liftEffect $ TS.convertTransaction input
-        let expected = TD.convertTransaction serialized
-        pure input `shouldEqual` hush expected
-      test "deserialization is inverse to serialization #3" do
-        let input = txFixture3
-        serialized <- liftEffect $ TS.convertTransaction input
-        let expected = TD.convertTransaction serialized
-        pure input `shouldEqual` hush expected
-      test "deserialization is inverse to serialization #4" do
-        let input = txFixture4
-        serialized <- liftEffect $ TS.convertTransaction input
-        let expected = TD.convertTransaction serialized
-        pure input `shouldEqual` hush expected
+    group "Transaction Roundtrips" do
```

Comment:
Please, wrap each of the fixtures in a `test` block. The point of adding many entries in the test is to immediately see which one is failing. Please undo. 

NB: I'm not against factoring out the function to test a single fixture.

Reply:
Fixed